storey/aggregations.py

```diff
@@ -685,6 +743,30 @@ def get_and_flush_pending(self):
         self.pending_aggr = {}
         return pending
 
+    def get_max_window_millis(self):
```

Comment:
Now im thinking maybe you should save the `max_windows`, `total_buckets` and `period` as a member of the class and calculate it once since it is used in a bunch of places. can impact performance.

Reply:
that's what i did in the previous pr :|